src/main/java/com/corundumstudio/socketio/handler/AuthorizeHandler.java

```diff
@@ -146,14 +146,29 @@ private boolean authorize(ChannelHandlerContext ctx, Channel channel, String ori
                                                 (InetSocketAddress)channel.remoteAddress(),
                                                     req.uri(), origin != null && !origin.equalsIgnoreCase("null"));
 
-        boolean result = false;
+        String redirectUrl = null;
         try {
-            result = configuration.getAuthorizationListener().isAuthorized(data);
+            redirectUrl = configuration.getAuthorizationListener().isRedirected(data);
+        } catch (Exception ignore) {
+        }
```

Comment:
It's a bad practice to swallow Exceptions. Please fix

Reply:
I'm not working on this PR branch anymore. I decided to carefully break compatibility in some places to implement more features, see https://github.com/mrniko/netty-socketio/pull/631 instead, where this is fixed.

Instead returning `isRedirected` in a separate @Override, I decided to move it into the same `authorize` function and return a response object instead a boolean. This breaks compatibility, but enables us to do a lot more.

I'd suggest you reject this PR and switch to the newer one.

Reply:
https://github.com/mrniko/netty-socketio/pull/631/commits/8fb13b0514acda7ff39b1d749080f410dfb3be27